optimum/intel/openvino/modeling_diffusion.py

```diff
@@ -217,9 +217,6 @@ def _from_pretrained(
                     cls.config_name,
                 }
             )
-            ignore_patterns = ["*.msgpack", "*.safetensors", "*pytorch_model.bin"]
-            if not from_onnx:
-                ignore_patterns.extend(["*.onnx", "*.onnx_data"])
```

Comment:
I think this bit was a fix from a previous PR to avoid downloading ONNX weights when the OV ones are available.

Reply:
Merged the main branch. Hope this is solved.

Reply:
Have you pushed?

I still see this PR branch reverting the fix: https://github.com/sawradip/optimum-intel/blob/integrate_timm/optimum/intel/openvino/modeling_diffusion.py#L229

Reply:
Now its fixed.   Right?

Reply:
It is, thank you!